toolkit/tools/internal/shell/shell.go

```diff
@@ -189,6 +189,25 @@ func ExecuteLiveWithCallback(onStdout, onStderr func(...interface{}), printOutpu
 	return
 }
 
+// ExecuteAndLogToFile runs a command in the shell and redirects stdout to the give file
+func ExecuteAndLogToFile(filepath string, command string, args ...string) {
+	cmd := exec.Command(command, args...)
+	outfile, err := os.Create(filepath)
+	if err != nil {
+		logger.Log.Errorf("Unable to create: %v", err)
```

Comment:
Please use `%s` instead of `%v` for error messages.

Reply:
updated